lib/beaker-answers/pe_conf.rb

```diff
@@ -82,6 +82,28 @@ def _generate_1_0_data
         pe_conf["#{ns}::puppetdb_host"] = puppetdb.hostname
       end
 
+      if any_host_with_role?('pe_postgres')
+        pe_conf["#{ns}::database_host"] = the_host_with_role('pe_postgres').hostname
+        if hosts[0].options[:pe_postgresql_options]
+          if hosts[0].options[:pe_postgresql_options][:security] == 'cert'
+            pe_conf["#{ns}::database_ssl"] = true
+            pe_conf["#{ns}::database_cert_auth"] = true
+          elsif hosts[0].options[:pe_postgresql_options][:security] == 'password'
+            pe_conf["#{ns}::activity_database_password"] = "PASSWORD"
```

Comment:
since we know have pe_postgresql_options, should we allow the password to be configurable?

Reply:
I don't have a real strong opinion on this. It could be added, but not sure on the reason for?